chapter_08_prettification.asciidoc

```diff
@@ -804,19 +808,21 @@ I think that looks great!
 .The lists page goes dark
 image::images/prettified-dark.png["Screenshot of lists page in dark mode. Cool."]
 
+//RITA: Whew. I was more concerned about the legibility of dark mode figures when printed, but thanks all the same.
 But it's very much a matter of personal preference,
 and my editor will kill me
```

Comment:
On second thought, would you consider changing this to "my editor will be mad at me" or even some other silly description (freak out, self-combust, explode, etc)? I'd rather not imply that we'll cause you harm.

Reply:
haha i don't think anyone would take it seriously, but sure i don't mind :)